strawberry/experimental/pydantic/object_type.py

```diff
@@ -216,13 +212,26 @@ def is_type_of(cls: Type, obj: Any, _info: GraphQLResolveInfo) -> bool:
         if has_custom_to_pydantic:
             namespace["to_pydantic"] = cls.to_pydantic
 
+        kwargs: Dict[str, object] = {}
+
+        # Python 3.10 introduces the kw_only param. If we're on an older version
+        # then generate our own custom init function
+        if sys.version_info >= (3, 10):
+            kwargs["kw_only"] = True
+        else:
+            kwargs["init"] = False
```

Comment:
I'm not 100% sure we need this, I need to double check

Reply:
looks like it is needed 😊 removing it will trigger this error: `TypeError: non-default argument 'other' follows default argument`